src/app.js

```diff
@@ -114,9 +115,10 @@ const timezone = 'America/New_York';
 
 const runJob = () => {
   try {
-    return updateGrantsGrantees();
+    // TTAHUB-263: Update Legacy Reason names.
+    return [updateGrantsGrantees(), updateReasonNames()];
```

Comment:
Instead of using the nightly cron to run it can we use the pattern for one time jobs from package.json, e.g. `populateLegacyNonGrantees `? This way we don't need to remove unnecessary jobs from the nightly cron and can better test it locally.

Reply:
I have updated this to use the same method as 'populateLegacyNonGrantees'. Can you have a look to make sure I didn't overlook something?